src/react-jw-player.jsx

```diff
@@ -43,6 +44,12 @@ class ReactJWPlayer extends Component {
       existingScript.onload = getCurriedOnLoad(existingScript, this._initialize);
     }
   }
+  shouldComponentUpdate(nextProps) {
+    return shouldComponentUpdate(this.props, nextProps);
+  }
+  componentDidUpdate() {
+    this._initialize();
```

Comment:
Is it necessary to run the full initialize method here again? Do we need to re-attach all the event listeners? And are we sure they aren't being applied twice by running it twice?